### ISSUE TITLE
Clean up happening after using the embed video feature?

### ISSUE THREAD

**Issue by Henning1984HH** [FreeTubeApp/FreeTube#98]
Hi and thank you for this great app!

I noticed that google analytics stuff might be loaded when the embedding of a youtube video inside FreeTube is used.

I don't know if these analytics files etc. get cached forever so that when you watch another video that google would be able to track you through stuff downloaded via embedding beforehand.

So if there is any kind of caching or temporary saving of files happening because of embedding, it should be cleaned at best when switching pages inside FreeTube and to be sure at app start again.

Keep up this great project!

TL:DR I opened a video via embedding and now I'm afraid I might be trackable inside FreeTube.

**Comment by PrestonN** [comment]
Hey there.  This is a good argument, I can understand if there are any concerns with this.  There are 2 things that I plan on changing with FreeTube in regards to this.  First off, cache will be cleared upon exiting FreeTube.  Secondly, YouTube actually provides an embeded video URL that isn't supposed to save cookies.  This changes the URL from `https://www.youtube.com/embed/VIDEOID` to `https://www.youtube-nocookie.com/embed/VIDEOID` and should prevent cookies from saving in the future.

If you are ever concerned with what is being saved by Electron / FreeTube, you can view the file structure on your machine.  Deleting files such as cache and cookies should be safe, but delete things at your own risk.

Here are the folder locations for FreeTube:

Windows
`C:\Users\USERNAME\AppData\Roaming\FreeTube\`

Mac
`/Users/USERNAME/Library/Application Support/FreeTube/`

Linux
`/home/USERNAME/.config/FreeTube/`

I will continue to work on removing things that might cause tracking through FreeTube.  I will go ahead and close this issue but if you see any more concerns about tracking then please let me know.
